### PR TITLE
[Doc] Add doc for `<BulkUpdateButton>`

### PR DESCRIPTION
**Pull request by slax57**

### CODE REVIEW

File: docs/Buttons.md [hunk 1]
```diff
@@ -246,11 +246,58 @@ export const PostList = () => (
...
+| `confirmContent`     | Optional | React node                              | -                  | Lets you customize the content of the confirm dialog. Only used in `'pessimistic'` or `'optimistic'` mutation modes |
+| `confirmTitle`       | Optional | `string`                                | -                  | Lets you customize the title of the confirm dialog. Only used in `'pessimistic'` or `'optimistic'` mutation modes   |
+| `label`              | Optional | `string`                                | 'ra.action.delete' | label or translation message to use                                                                                 |
+| `icon`               | Optional | `ReactElement`                          | `<DeleteIcon>`     | iconElement, e.g. `<CommentIcon />`                                                                                 |
+| `mutationMode`       | Optional | `'undoable'|'pessimistic'|'optimistic'` | `'undoable'`       | Mutation mode                                                                                                       |
```
Comment (djhi) [inline docs/Buttons.md:255]: This line break the table format

File: docs/Buttons.md [hunk 2]
```diff
@@ -246,11 +246,58 @@ export const PostList = () => (
...
+| `confirmContent`     | Optional | React node                              | -                  | Lets you customize the content of the confirm dialog. Only used in `'pessimistic'` or `'optimistic'` mutation modes |
+| `confirmTitle`       | Optional | `string`                                | -                  | Lets you customize the title of the confirm dialog. Only used in `'pessimistic'` or `'optimistic'` mutation modes   |
+| `icon`               | Optional | `ReactElement`                          | `<ActionUpdate>`   | An icon element                                                                                                     |
+| `label`              | Optional | `string`                                | 'ra.action.update' | Label or translation message to use                                                                                 |
+| `mutationMode`       | Optional | `'undoable'|'pessimistic'|'optimistic'` | `'undoable'`       | Mutation mode                                                                                                       |
```
Comment (djhi) [inline docs/Buttons.md:297]: Same

**Review by djhi — CHANGES_REQUESTED** [review]

File: docs/Buttons.md [hunk 3]
```diff
@@ -246,11 +246,58 @@ export const PostList = () => (
...
+    </List>
+);
+```
+{% endraw %}
+
+![Bulk Update button](./img/bulk-update-button.png)
+
+| Prop                 | Required | Type                                    | Default            | Description                                                                                                         |
+| -------------------- | -------- | --------------------------------------- | ------------------ | ------------------------------------------------------------------------------------------------------------------- |
+| `data`               | Required | `object`                                | -                  | An object with the fields that need to be updated on the selected records                                           |
```
Comment (adguernier) [inline docs/Buttons.md:292]: Should not we provide an example of the shape of the `data` object ?

Comment (slax57) [reply docs/Buttons.md:292]: You have one example in the snippet just above the table. Am I missing something?

**Review by djhi — COMMENTED** [review]
We should probably extract this in a dedicated documentation page and reference it there eventually

**Comment by slax57** [comment]
> We should probably extract this in a dedicated documentation page and reference it there eventually

Eventually yes, along with the other buttons referenced in `Buttons.md`. But for now I think simply documenting this (existing) button next to and in the same style as its siblings (`BulkDeleteButton` and `BulkExportButton`) is good enough.
Wdyt?

**Pull request merged**